ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -30,6 +30,10 @@ const user = require('component/singletons/user-instance');
 const cql = require('js/cql');
 const announcement = require('component/announcement');
 const SearchFormModel = require('component/search-form/search-form.js');
+const properties = require('properties')
+const lightboxResultInstance = require('component/lightbox/result/lightbox.result.view');
+const lightboxInstance = lightboxResultInstance.generateNewLightbox();
+const QueryResult = properties.hasExperimentalEnabled() ? require('component/result-form/result-form.view') : ''
```

Comment:
❓ I think it might be better to return empty objects (i.e. `{}`) for non-permitted `require` calls since I think `require` basically returns the `module.exports` export of the corresponding file.

Either case is basically a JS console log error waiting to happen if the code that uses that import isn't properly sectioned off, anyway, so I guess it doesn't ultimately matter as long as the code that uses that import never actually gets to run either